docs/labwc-config.5.scd

```diff
@@ -87,6 +89,15 @@ The rest of this man page describes configuration options.
 	*outlines* [yes|no] Draw an outline around the selected window when
 	switching between windows. Default is yes.
 
+*<windowSwitcher><fields><field content="" width="%">*
+	Define window switcher fields which can be any of:
+
+	*type* Show view type ("xdg-shell" or "xwayland")
+
+	*app_id* Show app_id (class for XWayland)
+
+	*title* Show window title if different to app_id
```

Comment:
Suggest adding a line like `*content* values can be one of:` and then the type, app_id and title explanations.

Do we want to document `width` being a percentage of whatever is the width of the OSD for completeness?

Reply:
Good point. Done.